### DISCUSSION TITLE
p/175a1706/

### DISCUSSION THREAD

**Discussion by giscus** [MHuiG/talk#9] [Giscus]
# p/175a1706/

如何在博客中添加简易文章推荐功能 - MHuiG - MHuiG

https://blog.mhuig.top/p/175a1706/

**Comment by xaoxuu** [comment]
厉害👍，可行的话就替代掉原来的那个插件吧

**Comment by W4J1e** [comment]
我还以为这篇文章会比较好懂一点，结果点进来看觉得还是一样难懂。
~。~
